bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/EnOceanBindingConstants.java

```diff
@@ -179,6 +184,30 @@
     public static final String CHANNEL_STATUS_REQUEST_EVENT = "statusRequestEvent";
     public static final String CHANNEL_SEND_COMMAND = "sendCommand";
 
+    public static final String CHANNEL_VENTILATIONOPERATIONMODE = "ventilationOperationMode";
+    public static final String CHANNEL_FIREPLACESAFETYMODE = "fireplaceSafetyMode";
+    public static final String CHANNEL_HEATEXCHANGERBYPASSSTATUS = "heatExchangerBypassStatus";
+    public static final String CHANNEL_SUPPLYAIRFLAPSTATUS = "supplyAirFlapStatus";
+    public static final String CHANNEL_EXHAUSTAIRFLAPSTATUS = "exhaustAirFlapStatus";
+    public static final String CHANNEL_DEFROSTMODE = "defrostMode";
+    public static final String CHANNEL_COOLINGPROTECTIONMODE = "coolingProtectionMode";
+    public static final String CHANNEL_OUTDOORAIRHEATERSTATUS = "outdoorAirHeaterStatus";
+    public static final String CHANNEL_SUPPLYAIRHEATERSTATUS = "supplyAirHeaterStatus";
+    public static final String CHANNEL_DRAINHEATERSTATUS = "drainHeaterStatus";
+    public static final String CHANNEL_TIMEROPERATIONMODE = "timerOperationMode";
+    public static final String CHANNEL_WEEKLYTIMERPROGRAMSTATUS = "weeklyTimerProgramStatus";
+    public static final String CHANNEL_ROOMTEMPERATURECONTROLSTATUS = "roomTemperatureControlStatus";
+    public static final String CHANNEL_AIRQUALITYVALUE1 = "airQualityValue1";
+    public static final String CHANNEL_AIRQUALITYVALUE2 = "airQualityValue2";
+    public static final String CHANNEL_OUTDOORAIRTEMPERATURE = "outdoorAirTemperature";
+    public static final String CHANNEL_SUPPLYAIRTEMPERATURE = "supplyAirTemperature";
+    public static final String CHANNEL_INDOORAIRTEMPERATURE = "indoorAirTemperature";
+    public static final String CHANNEL_EXHAUSTAIRTEMPERATURE = "exhaustAirTemperature";
+    public static final String CHANNEL_SUPPLYAIRFANAIRFLOWRATE = "supplyAirFanAirFlowRate";
+    public static final String CHANNEL_EXHAUSTAIRFANAIRFLOWRATE = "exhaustAirFanAirFlowRate";
+    public static final String CHANNEL_SUPPLYFANSPEED = "supplyFanSpeed";
+    public static final String CHANNEL_EXHAUSTFANSPEED = "exhaustFanSpeed";
+
     public static final Map<String, EnOceanChannelDescription> CHANNELID2CHANNELDESCRIPTION = Collections
```

Comment:
Use `Map.of` here

Reply:
As I have more than 10 entries here, I am using Map.ofEntries, hope this is the expected way